docs/io.cozy.contacts.md

```diff
@@ -41,19 +41,43 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
   - `label?`: {string} A user-provided localized type of instance
   - `primary?`: {boolean} Indicates a preferred-use instance
 - `relationships`: {object} links between documents
+
   - `groups`: {object} groups the contact belongs to
     - `data`: {array} list of groups
       - `_id`: {string} id of the io.cozy.contacts.groups document
-      - `_type`: {string} doctype "io.cozy.contacts.groups" 
-- `metadata`: {object}
-  - `version`: {integer} Used for migrations. Current version is **1**
+      - `_type`: {string} doctype "io.cozy.contacts.groups"
+  - `accounts`: {object}
+    - `data`: {array} list of accounts
+      - `_id`: {string} id of the io.cozy.contacts.accounts document
+      - `_type`: {string} doctype "io.cozy.contacts.accounts"
+
+- `cozyMetaData`: {object}
+
+  - `doctypeVersion`: {integer} Used for migrations. Current version is **2**
+  - `sourceAccount`: {string} id of the io.cozy.contacts.accounts object
```

Comment:
`sourceAccount: When the document was imported from a connector, identifier of the account in io.cozy.accounts`

(io.cozy.accounts will be the same as io.cozy.contacts.accounts but we should keep it generic cf https://github.com/cozy/cozy-doctypes/tree/master/docs#document-metadata)